techHttp.js

```diff
@@ -42,17 +43,43 @@ module.exports = function(mockRequest) {
 
     }
 
+    /**
+     * Simpler form of get
+     *
+     * @param options.url
+     * @param [options.category]
+     * @param [options.timeout]
+     * @param [options.auth]
+     * @param [options.headers]
+     *
+     * Options object is also passed as such to the 'request' library'
+     */
+    function getFromOpts(ruuid, opts) {
+        logger.info("[tech-http] Getting from opts", opts);
+        return wrapRequest(ruuid, "getAsync", _.extend({
+            json : true
+        }, opts));
+    }
+
     return {
         on: emitter.on.bind(emitter),
 
         // TODO [JLE] 'timeout' and 'auth' should be removed and declared instead directly in the given 'options' parameter
         get: function wrapGet(ruuid, url, category, timeout, auth, options) {
-            return wrapRequest(ruuid, "getAsync", _.extend({
-                url: url,
-                json: true,
-                auth: auth,
-                timeout: timeout
-            }, options), category);
+
+            if (!_.isString(url) && arguments.length === 2) {
+                var opts = url;
+                return getFromOpts(ruuid, opts);
+            } else {
+                return wrapRequest(ruuid, "getAsync", _.extend({
+                    url: url,
+                    json: true,
+                    auth: auth,
+                    timeout: timeout
+                }, options), category);
```

Comment:
The `category` parameter is ignored in the case a single `opts` parameter is specified. Are we OK with that, the value 'core' will always be used in this case ?